.github/workflows/test-model.yml

```diff
@@ -0,0 +1,59 @@
+name: test-model
+
+on:
```

Comment:
Does it make sense to do run this on push? New errors are probably not due to a code change, but to errors in cf-graphs / the currently running bot code? I think workflow_dispatch / cron triggers are better suited here

Reply:
Agree. I will change it to workflow_dispatch / cron triggers.